### DISCUSSION TITLE
Compact mode for the dashboard

### DISCUSSION THREAD

**Discussion by d4rken** [d4rken-org/sdmaid-se#1151] [Ideas]
Based on a [discord discussion](https://discord.com/channels/548521543039189022/1047079350157193236/1234315302188351498)

Exampleconcept from [a user]:

[SDMain_SE_concept](https://github.com/d4rken-org/sdmaid-se/assets/1439229/3a1cc3a6-d238-4cf4-ace2-94ee542ee08c)

Compact mode could be an opt-in feature for pro users.

Is there interest in this?

**Comment by freakingdan** [comment]
👋👋👋👋👋👋
